tests/conftest.py

```diff
@@ -312,15 +312,16 @@ def _subprocess_setup(request, cfg=None, uwsgi=False):
     from pykern import pkunit
     from pykern import pkio
     # different port than default so can run tests when supervisor running
-    p = '8002'
+    p = '8101'
```

Comment:
8100 should maybe be somewhere as a constant. Maybe we need a module sirepo.const that holds constants like this that can be imported without conflict.